routes/web.php

```diff
@@ -23,8 +24,15 @@
 Route::get('search/{path?}', function (Request $request, $path = null) {
     return redirect()->to('/places/' . $path . '?' . $request->getQueryString());
 });
-Route::get('places', 'GazetteerController@search')->name('places')->middleware('checkmaxpaging', 'cors'); 
-Route::get('places/{id?}', 'GazetteerController@search')->name('places'); //shows places with optional id, if no id is given it uses all results before applying filters
+Route::get('places/{path?}', function (Request $request ,  $path = null) {
+    $params = $request->all();
+    if (isset($params['id'])) {
+        return redirect()->to('/places/' . $params['id']);
+    }
+    return app('TLCMap\Http\Controllers\GazetteerController')->search($request, $path);
+});

```

Comment:
Can this logic be merged into `GazetteerController@search`?

Reply:
Seems like Route::get('places/{path?}', function (Request $request ,  $path = null) {
    return app('TLCMap\Http\Controllers\GazetteerController')->search($request, $path);
}); could work for all cases.

Reply:
I mean it's cleaner to let the `search` method from `GazetteerController` to handle this. Request to `places?id={id}` will go through this method. It makes sense if you can merge this redirect logic into the `search` method.

Reply:
And please DON'T overuse force commit. For things like this, it's OK to make incremental updates.

Reply:
Updated